include/apps/tools/OswAppCalculator.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <osw_hal.h>
+#include <osw_ui.h>
+#include "osw_app.h"
+
+
+class OswAppCalculator : public OswApp {
+  public:
+    OswAppCalculator(void) {
+        ui = OswUI::getInstance();
+    };
+    virtual void setup() override; 
+    virtual void loop() override; //checks the cursor position (iNum) and starts the appropriate function
+    virtual void stop() override;
+    void draw(); //draws the ui
+    void setPNum1(); //gets the input that tells if num1 is positive or negative
```

Comment:
Can you combine two functions that have similar functions into one? in a way that carries only parameters.
`setPNum1()` and `setPNum2()`

Reply:
And what is mean `P`?

Reply:
Hmm, @ich2a maybe just make those functions private - at that point we don't really have to care about their names anyways (as nobody else can use them) :)